packages/frontend-app/src/app/components/Select/Select.tsx

```diff
@@ -184,10 +184,8 @@ const Select: React.FC<IProps> = ({
         </div>
       )}
       <div className="relative" ref={dropdownSelectableAreaRef}>
-        <Button
-          variant="transparent"
-          htmlType="button"
-          className={classNames("select", {
+        <button
+          className={classNames("select button-transparent", {
```

Comment:
You don't need an extra classname i.e button-transparent. Apply your tailwind classes on the `select` classname

Reply:
noted